### DISCUSSION TITLE
Kata 4 ciclo FOR

### DISCUSSION THREAD

**Discussion by NayGL** [LaunchX-InnovaccionVirtual/CursoIntroPython#154] [Q&A] [answered]
![kata 4 ciclo for](https://user-images.githubusercontent.com/94994465/153686968-6f20db4d-bd9b-44c4-a57e-87402a3926d6.png)

Hola a todos!! Alguno que me pueda ayudar con este error? Me está costando muchísimo hacer cada una de las katas jejej no he pasado de la 4 pero seguiré intentando, se los agradezco!! :)

**Comment by Wilord20** [comment]
Creo que es porque a la lista de key_words le hace falta el corchete de cierre. Algo así:

key_words = ["average","temperature","distance"]

**Reply by NayGL** [reply]
Gracias!! @Wilord20

**Comment by jmonrealr** [comment]
En el print noto que falta un `)` de cierre, tambien hay un espacio antes de el, haciendo que no este indentado.
Tambien el corchete de cierre que mencionan arriba.

```Python
for sentence in text_parts:
    for key_word in key_words:
        if key_word in sentence:
            print(sentence.replace(' C', ' Celsius'))
            break
```

**Reply by NayGL** [reply]
Gracias!! @jmonrealr

**Comment by irvingfloley** [comment, marked as answer]
Te falta cerrar tu variable de key_words, debe de quedar algo así:

`key_words = ["average", "temperature", "distance"]`

De igual manera te falta cerrar bien la instrucción print al final ")"

`print(sentence.replace('C', 'Celsius'))`

Recuerda, los errores de sintaxis son de escritura en el código y se producen antes de la ejecución del mismo.

**Reply by NayGL** [reply]
Gracias!! @irvingfloley
